.github/file-paths.yaml

```
@@ -105,11 +108,15 @@ e2e_specs: &e2e_specs
   - "e2e/support/**"
   - "e2e/snapshot*/**"
 
+e2e_cross_version: &e2e_cross_version
+  - "e2e/test/scenarios/cross-version/**"
+
```

You can add more triggers here, but these two should be the bare minimum.

```suggestion
e2e_cross_version: &e2e_cross_version
  - ".github/workflows/e2e-cross-version.yml"
  - "e2e/test/scenarios/cross-version/**"

```